src/Reflection/ReflectionClass.php

```diff
@@ -931,7 +931,7 @@ public function getDefaultProperties(): array
     {
         return array_map(
             static fn (ReflectionProperty $property) => $property->getDefaultValue(),
-            array_filter($this->getProperties(), static fn (ReflectionProperty $property): bool => $property->isDefault()),
+            $this->getProperties(),
```

Comment:
Yes in BR. `ReflectionClass` is created only from AST, so it cannot have runtime properties. Runtime properties are implemented only in `ReflectionObject`: https://github.com/Roave/BetterReflection/blob/5.1.x/src/Reflection/ReflectionObject.php#L69

So the original code cannot be covered by `ReflectionClass` tests because the support is only in `ReflectionObject`.

Reply:
Thanks for clarifying!